[2024-04-01 15:46:05,436] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_training_pipeline.training_model manual__2024-04-01T15:45:52.666645+00:00 [queued]>
[2024-04-01 15:46:05,441] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_training_pipeline.training_model manual__2024-04-01T15:45:52.666645+00:00 [queued]>
[2024-04-01 15:46:05,441] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-04-01 15:46:05,441] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-04-01 15:46:05,441] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-04-01 15:46:05,530] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): training_model> on 2024-04-01 15:45:52.666645+00:00
[2024-04-01 15:46:05,539] {standard_task_runner.py:52} INFO - Started process 12603 to run task
[2024-04-01 15:46:05,547] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_scoring_training_pipeline', 'training_model', 'manual__2024-04-01T15:45:52.666645+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/lead_scoring_training_pipeline/.ipynb_checkpoints/lead_scoring_training_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmpz45qn7fz', '--error-file', '/tmp/tmpbho6z8r0']
[2024-04-01 15:46:05,548] {standard_task_runner.py:80} INFO - Job 36: Subtask training_model
[2024-04-01 15:46:05,665] {task_command.py:371} INFO - Running <TaskInstance: Lead_scoring_training_pipeline.training_model manual__2024-04-01T15:45:52.666645+00:00 [running]> on host ac2457c935b6
[2024-04-01 15:46:05,997] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_scoring_training_pipeline
AIRFLOW_CTX_TASK_ID=training_model
AIRFLOW_CTX_EXECUTION_DATE=2024-04-01T15:45:52.666645+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-04-01T15:45:52.666645+00:00
[2024-04-01 15:46:05,998] {logging_mixin.py:115} INFO - 2.6.0
[2024-04-01 15:46:06,003] {logging_mixin.py:115} INFO - Loading 'features' table
[2024-04-01 15:46:09,851] {logging_mixin.py:115} INFO - Loading 'target' table
[2024-04-01 15:46:10,099] {utils.py:187} INFO - Creating mlflow experiment
[2024-04-01 15:46:10,291] {logging_mixin.py:115} WARNING - /opt/conda/lib/python3.8/site-packages/sklearn/utils/validation.py:72 DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
[2024-04-01 15:46:10,382] {logging_mixin.py:115} INFO - [LightGBM] [Warning] bagging_fraction is set=0.9776942233774311, subsample=1.0 will be ignored. Current value: bagging_fraction=0.9776942233774311
[2024-04-01 15:46:10,383] {logging_mixin.py:115} INFO - [LightGBM] [Warning] feature_fraction is set=0.7938437977336682, colsample_bytree=1.0 will be ignored. Current value: feature_fraction=0.7938437977336682
[2024-04-01 15:46:10,383] {logging_mixin.py:115} INFO - [LightGBM] [Warning] bagging_freq is set=7, subsample_freq=0 will be ignored. Current value: bagging_freq=7
[2024-04-01 15:46:18,013] {logging_mixin.py:115} WARNING - Registered model 'LightGBM' already exists. Creating a new version of this model...
[2024-04-01 15:46:18,045] {logging_mixin.py:115} WARNING - 2024/04/01 15:46:18 INFO mlflow.tracking._model_registry.client: Waiting up to 300 seconds for model version to finish creation.                     Model name: LightGBM, version 2
[2024-04-01 15:46:18,045] {logging_mixin.py:115} WARNING - Created version '2' of model 'LightGBM'.
[2024-04-01 15:46:18,792] {logging_mixin.py:115} INFO - Precision= 0.7396136476087607
[2024-04-01 15:46:18,792] {logging_mixin.py:115} INFO - Recall= 0.7482259935591673
[2024-04-01 15:46:18,792] {logging_mixin.py:115} INFO - AUC= 0.7482259935591673
[2024-04-01 15:46:19,341] {logging_mixin.py:115} INFO - Inside MLflow Run with id 48570bc09f764a4a830305708f349003
[2024-04-01 15:46:19,408] {python.py:173} INFO - Done. Returned value was: None
[2024-04-01 15:46:20,361] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_scoring_training_pipeline, task_id=training_model, execution_date=20240401T154552, start_date=20240401T154605, end_date=20240401T154620
[2024-04-01 15:46:20,547] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-01 15:46:20,556] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
